dvc/objects/db/base.py

```diff
@@ -19,17 +19,33 @@ class ObjectDB:
     DEFAULT_CACHE_TYPES = ["copy"]
     CACHE_MODE: Optional[int] = None
 
-    def __init__(self, fs, path_info, **config):
+    def __init__(self, fs, path_info, tmp_dir, **config):
```

Comment:
`tmp_dir` isn't actually used anywhere in ODB yet, but it will be once the current remote index is moved out of Remote into ODB.

In the meantime, this is needed to be able to construct a Remote object from an ODB instance.